.travis.yml

```diff
@@ -8,14 +8,22 @@ matrix:
     osx_image: xcode10.3
     env:
       - MATRIX_EVAL="CC=clang && CXX=clang++ && PYTHON_VERSION=3.6.1 && PYTHON=python"
+      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip && LLVM_FILE=taichi-llvm-8.0.1.zip"
   - os: osx
     osx_image: xcode10.3
     env:
       - MATRIX_EVAL="CC=clang && CXX=clang++ && PYTHON_VERSION=3.7.1 && PYTHON=python"
+      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip && LLVM_FILE=taichi-llvm-8.0.1.zip"
   - os: osx
     osx_image: xcode10.3
     env:
       - MATRIX_EVAL="CC=clang && CXX=clang++ && PYTHON_VERSION=3.8.1 && PYTHON=python"
+      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/yuanming-hu/taichi_assets/releases/download/llvm8/taichi-llvm-8.0.1.zip && LLVM_FILE=taichi-llvm-8.0.1.zip"
+  - os: osx
+    osx_image: xcode10.3
+    env:
+      - MATRIX_EVAL="CC=clang && CXX=clang++ && PYTHON_VERSION=3.8.1 && PYTHON=python"
+      - LLVM_EVAL="LLVM_REMOTE_PATH=https://github.com/taichi-dev/taichi_assets/releases/download/llvm10/taichi-llvm-10.0.0-macos.zip && LLVM_FILE=taichi-llvm-10.0.0-macos.zip"
```

Comment:
IMHO we should use three LLVM 10 and one LLVM 8, since the latter is to be deprecated very soon.

Reply:
Reasonable. Let's hear what @yuanming-hu thinks about this.

Reply:
In release mode, all four builds will be compiled, and they will try to upload to PyPI. In Python 3.8, there're both LLVM 8 and LLVM 10 builds - whichever finishes early will be uploaded to PyPI. This creates a little randomness that we don't really want :-)

For simplicity, let's **build with LLVM 10** only in this PR. This means LLVM 8 is no longer officially maintained since no buildbot is using that anymore. We still keep the `#if LLVM_VERSION < 10` macros in the code so LLVM 8 will not be immediately broken, but if in the future if necessary we can simply drop LLVM 8 support. If someone complains, we can simply ask him to use LLVM 10 instead.

We have to cut LLVM 8 support sooner or later, and I think now is a good time. The developer installation doc should also be updated to remove LLVM 8.